crates/orchestrator/src/jobs/da_job/mod.rs

```diff
@@ -272,9 +280,31 @@ async fn state_update_to_blob_data(
         blob_data.push(*compiled_class_hash);
     }
 
+    // saving the blob data of the block to S3 bucket (if not running in test mod)
+    if !test_mode {
```

Comment:
test_mode shouldn't be a flag of a function that's used normally. ideally we can mock the S3 client so that when it's called nothing happens?